src/dataviews/category-dataview-model.js

```diff
@@ -22,7 +22,15 @@ module.exports = DataviewModelBase.extend({
   ),
 
   url: function () {
-    return this.get('url') + '?bbox=' + this.get('boundingBox') + '&own_filter=' + (this.get('enableFilter') ? 0 : 1);
+    var params = [];
+    if (this.get('boundingBox')) {
+      params.push('bbox=' + this.get('boundingBox'));
+    }
+    if (this.get('enableFilter')) {
+      params.push('own_filter=' + (this.get('enableFilter') ? 0 : 1));
```

Comment:
@alonsogarciapablo should this really be in a if-check? looks wrong compared to the prev code above…